src/api/parser/package.json

```diff
@@ -25,14 +28,18 @@
     "linkifyjs": "3.0.5",
     "normalize-url": "6.1.0",
     "rss-parser": "3.12.0",
-    "sanitize-html": "2.5.3"
+    "sanitize-html": "2.5.3",
+    "set-interval-async": "^2.0.3"
   },
   "engines": {
     "node": ">=14.0.0"
   },
   "devDependencies": {
+    "@senecacdot/eslint-config-telescope": "1.1.0",
     "babel-jest": "27.5.1",
     "env-cmd": "10.1.0",
-    "nodemon": "2.0.15"
+    "eslint": "7.32.0",
+    "nodemon": "2.0.15",
+    "supertest": "6.1.6"
```

Comment:
Question: why are these dependencies?

Reply:
`Nodemon` was already there, `eslint` is needed and I talked to Cindy before that `supertest` works well for eslint type of issues.

Reply:
Do you want me to delete the `supertest`?

Reply:
Does the 'parser' use it?

Reply:
No.

Reply:
You can remove it. The issue Cindy talked about was ESLint wanted to install `supertest` as a production dependency in some packages.